### ISSUE TITLE
8.1rc2 missing searchbar on header

### ISSUE THREAD

**Issue by libasys** [owncloud/contacts#961]
TestEnviro:
8.1rc2

Step to reproduce:
1. goto contacts
2. watch right side of the header

Expected result
you should see a search bar

Actual result 
no search bar is visible

**Comment by DeepDiver1975** [comment]
I'll bring back search for 8.2

**Comment by sabl0r** [comment]
Still missing in 8.2. Any update on this?

**Comment by aspdye** [comment]
Seems to be not implemented into the new version :( but why?

**Comment by piflixe** [comment]
I am really missing this feature too. Search field is displayed for a second while the site is loading, but then it disappears. I would really appreciate if it is reincluded in an upcoming update. 

Using Owncloud 8.1.3  with contacts app 0.4.0.0

Is the search field Owncloud or Contacts app functionality?

**Comment by aspdye** [comment]
I must honestly admit that I am quite surprised that you can not make it after more than a month that the calendar and contacts a search function (one of the most important functions for a calendar, and contacts) gets back. It is incomprehensible to me that you could publish this update at all (8.1) and it still does not could you fix with 8.2.

That this should only be fixed with 9.0, is not as good as the normal quality of ownCloud, because this will fix this problem first in February of next year. 

The use of 8.0 (or even 7.0) does not constitute a real alternative for some of us ;)

Calendar+ and Contacts+ (with which one after the update you could still search) are not longer in an really active development (Libasys hasn't realeased any updates since over an month).

But I have to say: I am more a man of ideas, support and marketing. Not for Coding. Therefore, it is difficult for me to assess the situation with calendar and contacts.

But I think we can not change anything here. Calendar and contacts are community projects in principle.

**Comment by jancborchardt** [comment]
@aspdye we know it’s unfortunate for the time being, but as you say Calendar and Contacts are community projects. There will already be massive improvements with the dav sync in core to make sure that’s stable. And then the app frontends themselves will also be updated.

**Comment by aspdye** [comment]
This Time the calendar gets refactored. Is the Same planend for Contacts Too?

**Comment by jancborchardt** [comment]
@aspdye yes, see https://github.com/Henni/contactsrework by @Henni @irgendwie etc. – help is very welcome. :)
This is because the codebase is quite old and with sync in core we only need the frontend. Hence it’s better to start with a new approach. @babelouest @jbtbnl would be cool if you join again. :)
